sql/src/test/java/io/crate/planner/operators/LogicalPlannerTest.java

```diff
@@ -299,7 +323,9 @@ private String printPlan(LogicalPlan plan) {
                 Order order = (Order) plan;
                 startLine("OrderBy[");
                 addSymbolsList(order.orderBy.orderBySymbols());
-                sb.append("]\n");
+                sb.append("]");
+                sb.append(Arrays.toString(order.orderBy.reverseFlags()));
+                sb.append(Arrays.toString(order.orderBy.nullsFirst())).append("\n");
```

Comment:
Could we make this more descriptive? `[false][null]` is kinda hard to read.

Reply:
We could probably use `OrderBy#explainRepresentation` to print it with ASC/DESC and NULLS FIRST / NULLS LAST